### PR TITLE
Add: abstract for 'The State of Fortran'

### PR DESCRIPTION
**Pull request by LKedward**
As discussed I will give a presentation based on the recent 'State of Fortran' collaborative paper as an introduction and overview to Fortran-lang and its activities. Edits and suggestions are welcome.

---

__cc.__ @aradi @certik @milancurcic @awvwgk @interkosmos @HaoZeke @scivision @aslozada @vmagnin @arjenmarkus @epagone @ivan-pi @everythingfunctional @hsnyder @brocolis @urbanjost @jvdp1 

Since the original paper is submitted, I will assume that all original authors are happy to be listed as co-authors here as well. __Please let me know if you would not like to be included.__

### CODE REVIEW

**Review by everythingfunctional — APPROVED** [review]
LGTM. Yes, I would like to be listed as an author. Thanks.

**Review by awvwgk — APPROVED** [review]
Looks good to me.

**Comment by HaoZeke** [comment]
Excellent, looks great and I look forward to it!

File: FortranCon2021-state-of-fortran/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Harris Snyder,
+Carlos Une,
+John Urban,
+Jérémie Vandenplas
+
+__Session:__ Fortran-lang minisymposium (10 min)
+
+__Abstract:__
+A new community of developers has formed to unite Fortran users and modernize the Fortran ecosystem.
+In this presentation, we describe the high-level features of Fortran that continue to make it a good choice for scientists and engineers in the 21st century and outline ongoing community efforts including the development of a Fortran standard library and package manager, the fostering of a friendly and welcoming online community, improved compiler support, and language feature development.
```
Comment (jvdp1) [inline FortranCon2021-state-of-fortran/README.md]:
```suggestion
In this presentation, we describe the high-level features of Fortran that continue to make it a good choice for scientists and engineers in the 21st century. We will also outline ongoing community efforts including the development of a Fortran standard library and package manager, the fostering of a friendly and welcoming online community, improved compiler support, and language feature development.
```
It was quite a long sentence. I tried to split it in 2 parts.

**Review by jvdp1 — APPROVED** [review]
LGTM. Thank you.

**Review by milancurcic — APPROVED** [review]
Thank you!

**Comment by epagone** [comment]
Happy to contribute. Thanks for including me.

**Comment by certik** [comment]
Looks good, thanks!

**Comment by aslozada** [comment]
Thanks for including me

**Comment by hsnyder** [comment]
Thanks for including me as well! Looks good to me.

**Comment by vmagnin** [comment]
Great! Thanks!

**Review by ivan-pi — APPROVED** [review]

**Comment by LKedward** [comment]
Thanks everyone - I've now submitted the abstract.

**Pull request merged**